src/Microsoft.Azure.SignalR.Common/Logging/MessageLog.cs

```diff
@@ -142,6 +142,18 @@ internal static class MessageLog
                 new EventId(130, "StartToCheckIfUserInGroup"),
                 StartToCheckIfUserInGroupTemplate);
 
+        private static readonly Action<ILogger, ulong?, string, Exception> _startToCloseConnection =
+                LoggerMessage.Define<ulong?, string>(
+                    LogLevel.Information,
+                    new EventId(140, "StartToCloseConnection"),
```

Comment:
better to use the same event name "StartToCloseConnection", need to keep the event name less. is no reason, call it sth like "normal close".